workflows/argo/scream-prognostic-run.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: scream-prognostic-run
+spec:
+  entrypoint: driver
+  volumes:
+  - name: gcp-credentials-user-gcp-sa
+    secret:
+      secretName: gcp-key
+  - name: data-volume
+    persistentVolumeClaim:
+      claimName: scream-storage-claim
+  - name: dshm
+    emptyDir:
+      medium: Memory
+  templates:
+  - name: driver
+    inputs:
+      parameters:
+      - {name: config, value: "/tmp/180.yaml"}
+      - {name: precompiled_case, value: "true"}
+      - {name: memory, value: "700Gi"}
```

Comment:
Will most SCREAM runs require the ultra sim pool? Just wondering since if not maybe use default memory and CPU for the highcompute pool.